ts/WoltLabSuite/Core/Api/Articles/MarkAllArticleAsRead.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Mark all articles as read
+ *
+ * @author Olaf Braun
+ * @copyright  2001-2025 WoltLab GmbH
+ * @license  GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
+ * @since 6.3
+ */
+
+import { prepareRequest } from "WoltLabSuite/Core/Ajax/Backend";
+import { ApiResult, apiResultFromError, apiResultFromValue } from "WoltLabSuite/Core/Api/Result";
+
+export async function markAllArticleAsRead(): Promise<ApiResult<[]>> {
```

Comment:
```suggestion
export async function markAllArticlesAsRead(): Promise<ApiResult<[]>> {
```
The filename has to be adjusted as well.